### ISSUE TITLE
Noexcept method vk::CommandBuffer::bindVertexBuffers throws exception

### ISSUE THREAD

**Issue by YaaZ** [KhronosGroup/Vulkan-Hpp#433]
I am getting following warning on MSVC:

> vulkan.hpp(65752): warning C4297: 'vk::CommandBuffer::bindVertexBuffers': function assumed not to throw an exception but does
vulkan.hpp(65752): note: __declspec(nothrow), throw(), noexcept(true), or noexcept was specified on the function

The method has `VULKAN_HPP_NOEXCEPT` macro and runtime assertion:
```c++
if ( buffers.size() != offsets.size() )
{
  throw LogicError( VULKAN_HPP_NAMESPACE_STRING "::VkCommandBuffer::bindVertexBuffers: buffers.size() != offsets.size()" );
}
```

**Comment by asuessenbach** [comment]
This is already resolved with PR #416

**Comment by YaaZ** [comment]
@asuessenbach Thank you! Closing the issue
